docs/guide-send-tx-bundle.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Sending Tx and Bundles
+---
+
+Flashbots provides a variety of methods for submitting transactions and bundles. This guide is designed to help you understand these options and select the one that best fits your needs.
+
+## Understanding Relay and RPC Endpoints
+
+Flashbots offers two primary JSON-RPC endpoints on Ethereum Mainnet: `rpc.flashbots.net` and `relay.flashbots.net`.
+
+### `rpc.flashbots.net` - For Retail Users
+- **Purpose**: Designed specifically for retail users as a drop-in RPC replacement in their wallet.
+- **Features**: Provides MEV protection and MEV refunds for eligible transactions. Support all the regular [Ethereum JSON RPC methods](https://docs.infura.io/networks/ethereum/json-rpc-methods/) but not [Flashbots specific JSON RPC
+methods](flashbots-auction/advanced/rpc-endpoint).
+- **Performance**: [Rate limited](/flashbots-protect/additional-documentation/ratelimiting) to satisfy the need of regular users.
+- **Note**: Transactions are submitted through `eth_sendRawTransaction` to fit wallet RPC interface. Underneath, the endpoint uses `relay.flashbots.net` to submit transactions.
+
+### `relay.flashbots.net` - For Advanced Users
+- **Purpose**: This endpoint is designed for advanced users, including searchers, applications, and Telegram bots. Its primary use is to submit transactions and bundles.
```

Comment:
Nit: The relay doesn't submit orderflow, it accepts it. Suggest changing last sentence to: "It accepts both transactions and bundles".